package.json

```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "pretest": "eslint --rulesdir=tools/eslint-rules lib test",
-    "test": "tap \"test/**/*.test.js\"",
+    "test": "tap test",
```

Comment:
Right now there's only one of those (`test/cli/start-cli.js`) which is a bit annoying. In future, I'd rather move those helpers into a sibling directory (`test-helpers/`) than risk the inconsistent glob behavior.